### DISCUSSION TITLE
How to view blogs?

### DISCUSSION THREAD

**Discussion by Little-swan** [TumblThreeApp/TumblThree#572] [Q&A (Question & Answer)]
I added the blog but if I click show files it's just a bunch of images and videos, none of the text is anywhere

**Comment by rachelombok** [comment]
I have the same question, did you ever figure this out? It is not clear from the site instructions. I just have .tumblr files and I am not sure how to open or view them.

**Comment by kristofersokk** [comment]
@Little-swan @rachelombok I also wanted to view, so I created a webpage which can visualize the blogs. Currently, I only support Tumblr blogs.
Github: https://github.com/kristofersokk/local-tumbl-viewer
Website: https://tumblviewer.app/
